h2o-docs/src/product/data-science/glm.rst

```diff
@@ -422,6 +422,12 @@ Multiclass Classification (Multinomial Family)
 
 Multinomial family generalization of the binomial model is used for multi-class response variables. Similar to the binomail family, GLM models the conditional probability of observing class "c" given "x". A vector of coefficients exists for each of the output classes. (:math:`\beta` is a matrix.) The probabilities are defined as:
 
+.. math::
+
+\hat{y}_c = Pr(y = c|x) = \frac{e^{x^\top\beta_c + \beta_{c0}}}{\sum^K_{k=1}(e^{x^\top\beta_k+\beta_{k0}})}
```

Comment:
Hello! You need to indent the math code right here for it to render properly please and thank you :)

Reply:
Hi, thanks for the comment, I missed this. It should be fixed now :)